src/quic/node_quic_crypto.cc

```diff
@@ -739,19 +802,39 @@ void InitializeTLS(const QuicSession& session) {
 
 void InitializeSecureContext(
     crypto::SecureContext* sc,
+    bool early_data,
     ngtcp2_crypto_side side) {
-  constexpr auto ssl_server_opts =
-      (SSL_OP_ALL & ~SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS) |
-      SSL_OP_SINGLE_ECDH_USE |
-      SSL_OP_CIPHER_SERVER_PREFERENCE |
-      SSL_OP_NO_ANTI_REPLAY;
+  // TODO(@jasnell): Should this be a static value?
+  constexpr static unsigned char session_id_ctx[] = "node.js quic server";
```

Comment:
My reading of the `SSL_CTX_set_session_id_context` is that this should be fine, but feel free to wait for confirmation from more people more familiar with crypto/tls :)

Reply:
For now, I'll leave it here but with a better todo message